packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -3475,6 +3482,7 @@ describe('NetworkController', () => {
                     url: `https://${infuraNetworkType}.infura.io/v3/{infuraProjectId}`,
                   },
                 ],
+                lastUpdatedAt: FAKE_DATE_NOW_MS,
```

Comment:
I like how we are now truthful with our tests and the actual state we are asserting.
I dislike how noisy it makes the assertions.

FUTURE (personal comment) - I would really like us to decouple and make an assertion function to reuse to reduce noise.

something like:
```
assertNetworkConfiguration(someNetworkConfiguration, { fieldsThatWeWantToInspectAndTest })
```

Meeeeh, I see pros and cons with this... And I at least prefer these strict object assertions over snapshots.